_posts/2024-XX-XX-quarkus-kiota.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: 'The road to generated SDKs with Kiota lands to Quarkus'
+date: 2024-XX-XX
+tags: release
+synopsis: 'Introducing a Quarkus-Kiota extension and available Java tools to work with Kiota.'
+author: andreaTP
+---
+
+== The Challenge
+:imagesdir: assets/images/posts/kiota
+
+Quarkus applications typically expose functionalities through API endpoints; here, we discuss how to easily consume those APIs and provide a smooth experience for our beloved users. While HTTP calls are easy to perform in any programming language and environment, when API complexity increases, and the number of available endpoints becomes uncountable, the need for better tools becomes apparent.
```

Comment:
nothing wrong in here but supposedly Quarkus already has solutions in the space of handling http client and server interactions. Kiota's "force" is the cross-language uniform support so maybe incorporate that earlier in here?